readme.md

```diff
@@ -691,7 +697,85 @@ coordsToPair(pairToCoords([1, 2])) // [1, 2]
 pairToCoords(coordsToPair({x: 1, y: 2})) // {x: 1, y: 2}
 ```
 
+### Homomorphism
 
+A homomorphism is just a structure preserving map. In fact, a functor is just a homomorphism between categories as it preserves the original category's structure under the mapping.
+
+```js
+A.of(f).ap(A.of(x)) == A.of(f(x))
```

Comment:
Perhaps also give simpler examples such as `x => x*2` preserving the addition structure?